src/histolab/slide.py

```diff
@@ -210,12 +210,12 @@ def name(self) -> str:
 
     @lazyproperty
     def processed_path(self) -> str:
-        """Retrieve the path to store processed files generated from the slide.
+        """Retrieve the path to store the tiles generated from the slide.
```

Comment:
I think that considering that this is a lazy property the docstring shouldn't contain the word Retrieve cause it doesn't retrieve anything it's a property